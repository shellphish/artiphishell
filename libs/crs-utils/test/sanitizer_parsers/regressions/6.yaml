focus_repo_path: "/src/assimp/"
language: c
sanitizer: address
exit_code: 77
expected_organizer_significance: 212 # RecognizedNonSanitizerNotableCrash
expected_organizer_crash_description: "LibFuzzer Generic crash and/or Jazzer Sanitizer Crash"
expected_parser: 'asan'
expected_summary: 'SUMMARY: libFuzzer: out-of-memory'
expected_sanitizer: libFuzzer
expected_crash_type: 'out-of-memory'
# expected_stack_trace_file_names:
#   main:
#     - source: ubsan_diag_standalone.cpp
#     - source: FuzzerUtil.cpp
#     - source: FuzzerLoop.cpp
#     - binary: libpthread.so.0
#     - binary: libc.so.6
#     - binary: libc.so.6
#     - source: AssertHandler.cpp
#     - source: DefaultIOSystem.cpp
#     - binary: assimp_fuzzer
#     - source: unzip.c
#     - source: unzip.c
#     - binary: assimp_fuzzer
#     - binary: assimp_fuzzer
#     - source: ZipArchiveIOSystem.cpp
#     - source: ZipArchiveIOSystem.cpp
#     - source: ZipArchiveIOSystem.cpp
#     - source: D3MFOpcPackage.cpp
#     - source: D3MFImporter.cpp
#     - source: Importer.cpp
#     - source: Importer.cpp
#     - source: assimp_fuzzer.cc
#     - source: FuzzerLoop.cpp
#     - source: FuzzerDriver.cpp
#     - source: FuzzerDriver.cpp
#     - source: FuzzerMain.cpp
#     - binary: libc.so.6
#     - binary: assimp_fuzzer