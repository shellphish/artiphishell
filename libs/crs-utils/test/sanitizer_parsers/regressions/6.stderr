+ '[' '!' -v TESTCASE ']'
+ '[' '!' -f /work/pov_input ']'
+ export RUN_FUZZER_MODE=interactive
+ RUN_FUZZER_MODE=interactive
+ export FUZZING_ENGINE=libfuzzer
+ FUZZING_ENGINE=libfuzzer
+ export SKIP_SEED_CORPUS=1
+ SKIP_SEED_CORPUS=1
+ run_fuzzer assimp_fuzzer /work/pov_input
vm.mmap_rnd_bits = 28
/out/assimp_fuzzer -rss_limit_mb=2560 -timeout=25 /work/pov_input < /dev/null
INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 3576381732
INFO: Loaded 1 modules   (235610 inline 8-bit counters): 235610 [0x555556fa54b8, 0x555556fded12), 
INFO: Loaded 1 PC tables (235610 PCs): 235610 [0x555556fded18,0x5555573772b8), 
/out/assimp_fuzzer: Running 1 inputs 1 time(s) each.
Running: /work/pov_input
Info,  T0: Load $$$___magic___$$$.
Info,  T0: Found a matching importer for this file format: Quake Mesh / 3D GameStudio Mesh Importer.
Info,  T0: Import root directory is './'
==16== ERROR: libFuzzer: out-of-memory (used: 2701Mb; limit: 2560Mb)
   To change the out-of-memory limit use -rss_limit_mb=<N>

SUMMARY: libFuzzer: out-of-memory